### PR TITLE
[Elevation] Add absoluteElevation property and clarify the elevationDidChange block param

### PR DESCRIPTION
**Pull request by yarneo**
As clients will initialize their material components and want to provide a background color that fits the component's elevation, they will need to get its absolute elevation value. Therefore, currently they need to add mdc_currentElevation + mdc_baseElevation themselves to get that.

In this PR I am adding an mdc_absoluteElevation getter where clients can get that and not have to do unneeded calculation in their code. Furthermore, because elevationDidChangeBlock effectively returns the absolute elevation, it would be easier for clients to understand that concept by also providing an absoluteElevation property.

Added easier to understand description of what exactly the elevationDidChangeBlock elevation param it provides.

### CODE REVIEW

File: components/Elevation/tests/unit/MDCMaterialElevationRespondingTests.m [hunk 1]
```diff
@@ -535,4 +535,19 @@ - (void)testElevationDidChangeForSuperviewBeforeSubviewsBlockIsCalled {
...
+  self.elevationOverrideViewController.mdc_overrideBaseElevation = fakeElevation;
+
+  // When
+  [self.elevationOverrideViewController.view addSubview:self.view];
+
+  // Then
+  XCTAssertEqualWithAccuracy(self.view.mdc_baseElevation, self.view.mdc_absoluteElevation,
+                             0.001);
+}
+
```
Comment (romoore) [inline components/Elevation/tests/unit/MDCMaterialElevationRespondingTests.m:593]: Please add tests for the varied containment/override arrangements as done for the other properties.

For example:

1.  ElevatableView -> OverrideView
2. OverrideView -> ElevatableView
3. ElevatableView -> View -> OverrideView
4. OverrideView -> View -> ElevatableView
5. ElevatableView -> OverrideView -> View

**Review by romoore — DISMISSED** [review]

File: components/Elevation/src/UIView+MaterialElevationResponding.h [hunk 2]
```diff
@@ -33,6 +33,14 @@
  */
 @property(nonatomic, assign, readonly) CGFloat mdc_baseElevation;
 
+/**
+ Returns the sum of the view's @c mdc_currentElevation with the @c mdc_currentElevation of its
```
Comment (codeman7) [inline components/Elevation/src/UIView+MaterialElevationResponding.h:37]: Nit: "Returns @c mdc_currentElevation plus @c mdc_baseElevation."

File: components/Elevation/src/MDCElevatable.h [hunk 3]
```diff
@@ -31,11 +31,12 @@
 
  Use this block to respond to elevation changes in the view or its ancestor views.
 
- @param elevation The @c mdc_currentElevation plus the @c mdc_currentElevation of all ancestor
- views.
+ @param absoluteElevation The @c mdc_currentElevation plus the @c mdc_currentElevation of all
```
Comment (codeman7) [inline components/Elevation/src/MDCElevatable.h:34]: Nit: "The @c mdc_currentElevation plus the @c mdc_baseElevation of the object. This equates to the @c mdc_absoluteElevation in @c MaterialElevationResponding."

**Review by codeman7 — APPROVED** [review]

File: components/Elevation/src/UIView+MaterialElevationResponding.h (hunk 2 above)
Comment (yarneo) [reply components/Elevation/src/UIView+MaterialElevationResponding.h:37]: this is written in the next line, do you still want me to change it?

File: components/Elevation/src/MDCElevatable.h (hunk 3 above)
Comment (yarneo) [reply components/Elevation/src/MDCElevatable.h:34]: mdc_baseElevation isn't a property in this protocol, i tried to explain it using what this protocol provides to make it easier on the user. Do you still want me to update to your copy?

File: components/Elevation/tests/unit/MDCMaterialElevationRespondingTests.m (hunk 1 above)
Comment (yarneo) [reply components/Elevation/tests/unit/MDCMaterialElevationRespondingTests.m:593]: done

File: components/Elevation/src/UIView+MaterialElevationResponding.h (hunk 2 above)
Comment (yarneo) [reply components/Elevation/src/UIView+MaterialElevationResponding.h:37]: talked offline, resolving.

File: components/Elevation/src/MDCElevatable.h (hunk 3 above)
Comment (yarneo) [reply components/Elevation/src/MDCElevatable.h:34]: talked offline, resolving.

**Pull request merged**